src/deepsparse/pipeline.py

```diff
@@ -176,29 +177,33 @@ def __init__(
 
         self._batch_size = self._batch_size or 1
 
-    def __call__(self, *args, **kwargs) -> BaseModel:
+    def __call__(self, *args, monitoring: bool = False, **kwargs) -> BaseModel:
         if "engine_inputs" in kwargs:
             raise ValueError(
                 "invalid kwarg engine_inputs. engine inputs determined "
                 f"by {self.__class__.__qualname__}.parse_inputs"
             )
+        timer = TimingBuilder()
```

Comment:
Have you tested this in the server with multiple requests happening concurrently? This is local to this function, so I _think_ we shouldn't have interference from other requests? I'm not even really sure how to tell if there is interference

Reply:
this was the intended effect of keeping the builder local to this function, we can look more into this after we integrate with the server